dashboard/app/models/course_offering.rb

```diff
@@ -211,6 +221,19 @@ def summarize_for_edit
     }
   end
 
+  def summarize_for_catalog
+    {
+      key: key,
+      display_name: display_name,
+      grade_levels: grade_levels,
+      image: image,
+      cs_topic: cs_topic,
+      school_subject: school_subject,
+      device_compatibility: device_compatibility,
+      course_version_path: path_to_latest_published_version
+    }
+  end
```

Comment:
What does replacing serialize with summarize_for_catalog do here? I saw above in the changes to curriculumCatalogShape.jsx, several props were removed (similar to summarize_for_catalog having fewer fields than serialize) since they aren't being used which makes sense, but won't we need to use them in the future?

Reply:
Good call –– I'm under the impression that we don't need them, but I'm realizing device compatibility is needed. Are there others that will be needed?

Part of me wonders if we should add attributes as we go (keeping it as minimal as possible until we need more), but if it's clear we really will need every single field, I agree it doesn't make sense to create a separate category.

Reply:
The big thing is now we have an attribute (course_version_path) that is not a serialized field. Do you think that field should be serialized too?

Reply:
Oh I'm seeing device compatibility is still in there 😅

Reply:
I think it would be good to also have it be serialized if possible!

Reply:
It feels risky to have it serialized because if the latest published version of the course offering changes, the link would change here too –– and then we'd have two sources of truth (one in the DB and one in the JSON). What is your reasoning for ensuring it's serialized?

Reply:
Ohhhhh I see, yes you're right good call. Okay I feel comfortable with this then! Thank you for walking me through that!

Reply:
+1 Meg, thanks for explaining! This makes sense to me